code/game/gamemodes/game_mode.dm

```diff
@@ -120,15 +163,39 @@
 		replacementmode.make_antag_chance(character)
 	return
 
+/datum/game_mode/proc/make_special_antag_chance(mob/living/character)
+	if(character.mind.antag_datums)
+		if(character.mind.antag_datums.len > 0)
+			return
+	//Check if they are banned
+	if(is_banned_from(character.ckey, list(ROLE_SPECIAL)) || QDELETED(character))
+		return
+	for(var/datum/special_role/subantag in active_specials)
+		if(!subantag.latejoin_allowed)
+			continue
+		if(subantag.spawn_mode == "midround")

```

Comment:
> string comparisons

😩

Reply:
Use defines instead if possible.

Reply:
Using a define instead, before my define wasn't recognised in gamemode, but I put it in the correct place this time 🤦